### ISSUE TITLE
Clarification on LAB-to_RGB converter

### ISSUE THREAD

**Issue by barchettamagica** [Mister-Dinky/LAB-to-RGB#1]
I need to convert LAB to RBG and color the cell next to it. I tried your LAB-to-RGB but I don't understand how to use it, I'm not very confident with VBA, can you explain it in an easy steb by step? I would really appreciate that! Thank you

**Comment by Mister-Dinky** [comment]
Ofcourse, happy to help :)

The Range Selection box on the left is used for the cells with the LAB values (one value per column), the Range Selection box on the right for the output (Which is usually three columns with the same amount of rows for RGB values). The tool will also color the cells for you if you keep that checkbox checked. The settings at the top are for which illuminant you want to use and if you want some of the parameters to be either the calculated one or the Standard ones (they give as good as the same result).

In short:
- Select the input LAB values in the left Range Box, (x rows by 3 columns).
- Select the output RGB range with the right Range Box (the same amount of rows by 3 columns)
- click on the button to start the conversion.

**Comment by haldunkocak** [comment]
Hi Mister-Dinky,

I would like to thank you for your sharing such a useful codes for others and I appreciate your effort about it. 
I need some help about codes. I tried to implement them into Visual Basic, then when I run the macro, whatever I do, I get warning which says "**Compile error ; Invalid outside procedure**" . Could you help me about running this macro in excel. 

Thanks a lot.
All bests
